src/lib/stores/workstreams/methods/fetchRelevantWorkstreams.ts

```diff
@@ -1,13 +1,18 @@
 import { workstreamsStore } from '..';
 
-export default async function (address: string) {
+export default async function (address: string, chainId: number) {
   return await Promise.all([
     await workstreamsStore.getWorkstreams({
-      assignedTo: address
+      assignedTo: address,
+      chainId: String(chainId)
     }),
     await workstreamsStore.getWorkstreams({
-      createdBy: address
+      createdBy: address,
+      chainId: String(chainId)
     }),
-    await workstreamsStore.getWorkstreams({ applied: 'true' })
+    await workstreamsStore.getWorkstreams({
+      applied: 'true',
+      chainId: String(chainId)
```

Comment:
Why do we convert to string here?

Reply:
Good question, no particular reason other than this gets sent as URL params, which are always strings. Agree it'd be better to be able to set this as an int though. Will adjust